GameServerLib/Logic/Packets/PacketHandlers/Handlers/HandleKeyCheck.cs

```diff
@@ -41,15 +51,15 @@ public bool HandlePacket(Peer peer, byte[] data)
                     player.Peer = peer;
                     var response = new KeyCheck(keyCheck.userId, playerNo);
                     bool bRet = _game.PacketHandlerManager.sendPacket(peer, response, Channel.CHL_HANDSHAKE);
-                    //handleGameNumber(player, peer, _game);//Send 0x91 Packet?
+                    //HandleGameNumber(player, peer, _game);//Send 0x91 Packet?
```

Comment:
I am leaving this there in case it turns out we need to send this packet. My best guess is this comes from older versions of league where it was necessary.